### PR TITLE
MMB: Add markers to sector map indicating all stop for the booking

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js [hunk 1]
```diff
@@ -3,25 +3,96 @@
 import * as React from 'react';
 import { MapView } from '@kiwicom/margarita-map';
 import { StyleSheet } from '@kiwicom/universal-components';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import { getCenter } from 'geolib';
+import memoize from 'memoize-one';
+import stringify from 'json-stable-stringify';
```
Comment (RobinCsl) [inline apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js]: I think I read about this somewhere on Slack, right? This is to make sure the stringified version is always ordered in the same way, right?

File: apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js [hunk 2]
```diff
@@ -3,25 +3,96 @@
 import * as React from 'react';
 import { MapView } from '@kiwicom/margarita-map';
 import { StyleSheet } from '@kiwicom/universal-components';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+import { getCenter } from 'geolib';
```
Comment (RobinCsl) [inline apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js]: Is geolib also used by FE?

File: apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js [hunk 3]
```diff
@@ -3,25 +3,96 @@
...
+    this.map = ref;
+  };
+
+  getMappedCoordinates = () => {
+    return mapCoordinates(this.props.data?.segmentLocations);
+  };
+
+  fit = () => {
+    if (this.map != null) {
+      // $FlowExpectedError: fitToCoordinates does exist on map, but flow is confused by .native|.web exports
```
Comment (RobinCsl) [inline apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js]: I have a suspicion that `react-native-web` flow-typed definitions are the reason behind many of these false positives.

File: apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js (hunk 1 above)
Comment (tbergq) [reply apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js]: yes [docs here](https://github.com/substack/json-stable-stringify#json-stable-stringify)

File: apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js (hunk 2 above)
Comment (tbergq) [reply apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js]: 🤷‍♂️ it is used in kiwicom/mobile

File: apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js (hunk 3 above)
Comment (tbergq) [reply apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js]: I suspect [this one](https://github.com/kiwicom/margarita/blob/master/packages/map/src/MapView.web.js)

**Pull request merged**